webpack.config.js

```diff
@@ -17,23 +17,28 @@
 const path = require('path');
 
 module.exports = {
-  entry: './src/fallback.ts',
+  target: 'node',
+  entry: './src/index.ts',
   output: {
-    library: 'Gax',
-    filename: './gax.js',
+    filename: './gax-bundled.js',
+    path: path.resolve(__dirname, 'build/src'),
+    libraryTarget: 'commonjs',
   },
   resolve: {
     extensions: ['.ts', '.js', '.json'],
```

Comment:
Are externals bundled as well? Is there a reason to list them in dependencies then?

Reply:
It puts all the dependencies from `node_modules` to a bundle. Is this what you're asking?

Reply:
Yep. So whole "dependencies" list can be removed, isn't it?